transports/janus_http.c

```diff
@@ -1108,6 +1108,7 @@ int janus_http_handler(void *cls, struct MHD_Connection *connection, const char
 	if (!strcasecmp(method, "OPTIONS")) {
 		response = MHD_create_response_from_buffer(0, NULL, MHD_RESPMEM_PERSISTENT);
 		MHD_add_response_header(response, "Access-Control-Allow-Origin", "*");
+    MHD_add_response_header(response, "Access-Control-Max-Age", "86400");
```

Comment:
Broken indentation, and same for other lines you added. We use tabs, not spaces.